src/Linter/Sniff/Visitor/ConfigNoCacheVisitor.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace Helmich\TypoScriptLint\Linter\Sniff\Visitor;
+
+use Helmich\TypoScriptLint\Linter\Report\Issue;
+use Helmich\TypoScriptLint\Linter\Sniff\ConfigNoCacheSniff;
+use Helmich\TypoScriptParser\Parser\AST\Operator\Assignment;
+use Helmich\TypoScriptParser\Parser\AST\Statement;
+
+class ConfigNoCacheVisitor implements SniffVisitor
+{
+    /** @var Issue[] */
+    private $issues = [];
+
+    /**
+     * @return Issue[]
+     */
+    public function getIssues(): array
+    {
+        return $this->issues;
+    }
+
+    public function enterTree(array $statements): void
+    {
+    }
+
+    public function enterNode(Statement $statement): void
+    {
+         if (!$statement instanceof Assignment) {
+             return;
+         }
+         if ($statement->object->relativeName !== 'no_cache'
+             && !str_ends_with($statement->object->relativeName, '.no_cache')) {
```

Comment:
[`str_ends_with`](https://www.php.net/manual/en/function.str-ends-with.php) was only introduced with PHP 8. I'd prefer keeping the PHP compatibility of this project as-is, for now.

```suggestion
             && substr($statement->object->relativeName, -9) !== '.no_cache') {
```

Reply:
I saw there was a polifill in your composer dependencies, but sure can be changes

Reply:
:+1: Thanks! I overlooked that the PHP 7 pipelines were obviously passing already 🙈. Still, that polyfill library was only a transitive dependency (of some of the Symfony libraries, presumably) and should have been declared explicitly in this project's `composer.json` to be safe.